lib/modules/downref.mjs

```diff
@@ -163,7 +163,9 @@ function normalizeDraftReferences (references) {
   return references
     .map((ref) => {
       let normalized = ref.replace(/^\[|\]$/g, '')
-      normalized = normalized.replace(/-\d{2}$/, '')
+      normalized = normalized
+        .replace(/-\d{2}$/, '')
```

Comment:
If this code ever encounters a reference to document without its version number (which is a common thing for _other_ software here to have to do - it means "go find the current version of the draft with this name") it will fail.

Consider draft-ietf-tsvwg-ieee-802-11-11.txt: The unversioned reference would be to `I-D.ietf-tsvwg-ieee-802-11`. This code would then try to look up `ietf-tsvwg-ieee-802` and fail (the correct lookup would be for `draft-ietf-tsvwg-ieee-802-11`).

I need to review more and search for examples before approving this or requesting changes